sway/tree/container.c

```diff
@@ -1609,3 +1619,68 @@ bool container_is_scratchpad_hidden_or_child(struct sway_container *con) {
 	con = container_toplevel_ancestor(con);
 	return con->scratchpad && !con->workspace;
 }
+
+static enum sway_container_layout layout_category(enum sway_container_layout layout) {
```

Comment:
Nit: `container_...` naming scheme to match the rest of this file.

Reply:
I replaced this helper with a new helper function "is_parallel" similar to commands/move